### ISSUE TITLE
Document pinout

### ISSUE THREAD

**Issue by probonopd** [johnnyr/Mini-Rambo#5]
Please document the pinout of all of the headers.

**MiniRambo1.3a**
![pinout](https://user-images.githubusercontent.com/2480569/28038775-4872064e-65c0-11e7-9c6a-46cb7ff6c48c.png)

**Comment by urbaniak** [comment]
That's the only one schema I've found describing the p3 (yellow) connector, looks great!

Got one question - does the RX/TX pins there are the same serial interface as the one available via usb? I want to connect octopi on raspberry (zero maybe) directly to the board (not via usb, which causes board reset when connecting).

**Comment by probonopd** [comment]
No, these RX/TX pins on connector P3 (SERIAL_PORT 2) are not the same pins as the one connected to USB. According to the Ultimachine technical support, the RX/TX connected to USB (SERIAL_PORT 0) is only available as test points on the back side of the board. Bummer.
